beesdoo_print_label/views/report_pricetag_normal.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <template id="css_qweb_template_pricetag_normal" inherit_id="web.report_assets_common">
+        <xpath expr="." position="inside">
+            <link href="/beesdoo_print_label/static/css/pricetag_base.scss" rel="stylesheet" type="text/scss"/>
+            <link href="/beesdoo_print_label/static/css/pricetag_normal_base.scss" rel="stylesheet" type="text/scss"/>
+            <link href="/beesdoo_print_label/static/css/pricetag_normal_normal.scss" rel="stylesheet" type="text/scss"/>
+        </xpath>
+    </template>
+
+    <template id="qweb_template_pricetag_normal">
+        <t t-call="web.basic_layout">
+            <t t-foreach="category_data['lines']" t-as="line">
+                <t t-foreach="line.quantity" t-as="q">
+                    <t t-set="product_tmpl_id" t-value="line.product_id.product_tmpl_id"/>
+                    <div class="normal_pricetag normal_pricetag_normal floating_box">
+                        <div class="product_name">
+                            <t t-esc="product_tmpl_id.name"/>
+                        </div>
+                        <div class="product_informations">
+                            <t t-if="product_tmpl_id.main_seller_id.name">
```

Comment:
What happens if there is no `main_seller_id`? Shouldn't you rather test on `product_tmpl_id.main_seller_id`?

Reply:
Without main_seller_id, the recordset is empty and the condition is false.

Reply:
👌